test/extensions/stats_sinks/hystrix/hystrix_test.cc

```diff
@@ -17,6 +19,7 @@ using testing::Invoke;
 using testing::NiceMock;
 using testing::Return;
 using testing::_;
+using Envoy::Server::Configuration::AdminStreamMock;
```

Comment:
nit: in the OSS Envoy world, we generally only use using declarations for elements in `testing::`.